examples/benchmarks/image_classification_cinic10.ipynb

```diff
@@ -2,39 +2,57 @@
  "cells": [
```

Comment:
<p>Delete this output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>